libkbfs/tlf_handle.go

```diff
@@ -626,8 +632,29 @@ func (pr partialResolver) Resolve(ctx context.Context, assertion string) (
 // resolved except for unresolved assertions in other; this should
 // equal other if and only if true is returned.
 func (h TlfHandle) ResolvesTo(
-	ctx context.Context, codec Codec, resolver resolver, other *TlfHandle) (
+	ctx context.Context, codec Codec, resolver resolver, other TlfHandle) (
 	resolvesTo bool, partialResolvedH *TlfHandle, err error) {
+
+	// Check the conflict extension.
+	var conflictAdded, finalizedAdded bool
+	if h.conflictInfo == nil && other.conflictInfo != nil {
```

Comment:
Maybe `TlfHandle` should get `IsConflict()` and `IsFinal()` methods too, like `BareTlfHandle`?